structuralalignment/superposition/theseus.py

```diff
@@ -215,9 +185,21 @@ def _parse_superposition(self, output):
         ----------
         output : bytes
         """
-        return output
+        for line in output.splitlines():
+            if "Classical" in line:
+                rmsd = line.split()[5]
+        return {"rmsd": rmsd}
```

Comment:
Include the other keys as well, even if they are empty.
```suggestion
        return {"rmsd": rmsd, "metadata": {}, ...}
```